src/pages/ActivationPage.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Route } from 'react-router-dom'
+
+import { render } from '../../test/testUtils'
+
+import ActivationPage from './ActivationPage'
+
+// https://www.npmjs.com/package/jest-fetch-mock
+import fetch, { FetchMock, GlobalWithFetchMock } from 'jest-fetch-mock'
+const customGlobal: GlobalWithFetchMock = global as GlobalWithFetchMock
+customGlobal.fetch = fetch as FetchMock
+customGlobal.fetchMock = customGlobal.fetch as FetchMock
+
+it(`fetches the card data after 1 second`, async () => {
+  jest.useFakeTimers()
+
+  fetchMock.mockResponses(
+    [JSON.stringify({}), { status: 200 }],
+    [JSON.stringify({ card: 'VX.precinct-21.5R' }), { status: 200 }],
+    ['', { status: 500 }]
+  )
+
+  render(<Route path="/" component={ActivationPage} />, {
+    route: '/',
+  })
+
+  expect(setInterval).toHaveBeenCalledTimes(1)
```

Comment:
Let's avoid naked globals. Please change to `expect(window.setInterval).toHaveBeenCalledTimes(1)`